bugbot/rules/performancebug.py

```diff
@@ -0,0 +1,101 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from bugbot.bugbug_utils import get_bug_ids_classification
+from bugbot.bzcleaner import BzCleaner
+from bugbot.utils import nice_round
+
+
+class PerformanceBug(BzCleaner):
+    def __init__(self):
+        super().__init__()
+        self.autofix_bugs = []
+
+    def description(self):
+        return "[Using ML] Bugs with Missing Performance Impact"
+
+    def columns(self):
+        return ["id", "summary", "confidence", "autofixed"]
+
+    def get_bz_params(self, date):
+        start_date, _ = self.get_dates(date)
+
+        params = {
+            "include_fields": ["id", "summary"],
+            "f1": "creation_ts",
+            "o1": "greaterthan",
+            "v1": start_date,
+            "f2": "keywords",
+            "o2": "nowords",
+            "v2": "perf,topperf,main-thread-io",
+            "f3": "cf_performance_impact",
+            "o3": "equals",
+            "v3": ["---"],
+        }
+
+        return params
+
+    def get_bugs(self, date="today", bug_ids=[]):
+        # Retrieve the bugs with the fields defined in get_bz_params
+        raw_bugs = super().get_bugs(date=date, bug_ids=bug_ids, chunk_size=7000)
+
+        if len(raw_bugs) == 0:
+            return {}
+
+        # Extract the bug ids
+        bug_ids = list(raw_bugs.keys())
+
+        # Classify those bugs
+        bugs = get_bug_ids_classification("performancebug", bug_ids)
+
+        results = {}
+
+        for bug_id in sorted(bugs.keys()):
```

Comment:
Why do we need to sort them here? They will be sorted later based on the order of the columns.